docs/real-time_wide_table.md

```diff
@@ -63,44 +62,41 @@ CREATE TABLE `order_wide_table`
 );
 ```
 
-Using flink sql client to sink in `Real-time wide table` scenario. The `mock_${value}` will never be used，just a placeholder:
+Using flink sql client to sink in `Real-time wide table` scenario. You can set an arbitrary value to the column which does not exist in `'tidb.sink.update-columns'`, since they will never be used, just as a placeholder:
```

Comment:
Could you give a link or some context about why setting an arbitrary here? Maybe there is any confusion for the reader in using this function.

Reply:
it's under below

Reply:
see line 90~93